.github/workflows/ci.yaml

```diff
@@ -406,3 +406,28 @@ jobs:
         with:
           name: release-images
           path: ./release-images
+
+  trivy:
+    runs-on: ubuntu-18.04
```

Comment:
❓ I don't know if it can use `ubuntu-latest` - their `README.md` was updated recently and uses this version so I stuck with it

Reply:
Actions generally are just small Node.js/JavaScript programs. GitHub's Actions Runner includes Node.js 12, so these actions just do something like run commands to download trivy and run the scan. So my guess is that using a more recent version of Ubuntu should be fine, though I would suggest pinning it to `ubuntu-20.04` because otherwise GitHub issues a warning about ubuntu-latest changing from ubuntu-18.04 to ubuntu-20.04

It just generally seems better for us to control versions of things so that we can decide when to upgrade, and avoid breakages

Reply:
Ahh, okay that makes more sense. I'll update to `ubuntu-20.04` then. Thanks for the explanation! 🙌